src/renderers/widget.ts

```diff
@@ -338,37 +338,49 @@ class RenderedPDF extends Widget {
 export
 function handleUrls(node: HTMLElement, resolver: RenderMime.IResolver, pathHandler: RenderMime.IPathHandler | null): Promise<void> {
   let promises: Promise<void>[] = [];
-  let imgs = node.getElementsByTagName('img');
-  for (let i = 0; i < imgs.length; i++) {
-    promises.push(handleImage(imgs[i], resolver));
+  // Handle HTML Elements with src attributes.
+  let nodes = node.querySelectorAll('*[src]');
+  for (let i = 0; i < nodes.length; i++) {
+    promises.push(handleSource(nodes[i] as HTMLImageElement, resolver));
```

Comment:
Also, can you update the documentation:

```
  * Resolve relative urls in elements
```
and maybe a note says it handles elements with src attributes, href, and link tags.